floss/render/default.py

```diff
@@ -221,7 +225,7 @@ def render_decoded_strings(decoded_strings: List[DecodedString], ostream, verbos
                 ostream.writeln("\n")
 
 
-def render_heading(heading, n, ostream, disable_headers):
+def render_heading(heading, n, ostream, disable_headers, verbose):
```

Comment:
nit: `verbose` argument before `disable_headers` to be consistent with other functions